frontend/src/components/shared/WithItemRenderComponent.tsx

```diff
@@ -79,7 +79,9 @@ function WithItemRenderComponentFactory<W extends WithItemFunc, T extends FMSTra
             return render(object)
         } else {
             if (placeholder) {
-                return placeholder
+                if (typeof placeholder === 'string') {
+                    return <span>placeholder</span>
+                } else return placeholder
```

Comment:
Doesn't just returning placeholder work? Because `string` should be part of `ReactNode`.

Reply:
TBH, I don't remember why I wrapped the string with a span. I think I ran into a problem somewhere and the span fixed it.